checker-framework-eclipse-update-site/manual/index.html

```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
```

Comment:
Is this manual for a previous version of the Eclipse plug-in?
Did you adapt the instructions to the new plug-in behavior?
You should also attribute sources, e.g. in a README file.

Reply:
Removed the manual folder.